rholang/src/test/scala/coop/rchain/rholang/interpreter/CostAccountingReducerTest.scala

```diff
@@ -84,13 +84,9 @@ class CostAccountingReducerTest extends FlatSpec with Matchers with TripleEquals
     implicit val errorLog = new ErrorLog[Task]()
     implicit val rand     = Blake2b512Random(128)
     implicit val cost     = CostAccounting.initialCost[Task](Cost(1000)).runSyncUnsafe(1.second)
-    val reducer = new DebruijnInterpreter[Task, Task.Par](
-      tuplespaceAlg,
-      Map.empty,
-      Ref.unsafe[Task, DeployParameters](DeployParameters.empty)
-    )
-    val send = Send(Par(exprs = Seq(GString("x"))), Seq(Par()))
-    val test = reducer.inj(send).attempt.runSyncUnsafe(1.second)
+    val reducer           = new DebruijnInterpreter[Task, Task.Par](tuplespaceAlg, Map.empty)
```

Comment:
I think the changes here belong to a different commit.

Reply:
Nope, it is the right one :)